t/torchvision/torchvision_ubi_9.3.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash -e
+
+# -----------------------------------------------------------------------------
+#
+# Package           : vision
+# Version           : v0.16.1
+# Source repo       : https://github.com/pytorch/vision.git
+# Tested on         : UBI:9.3
+# Language          : Python
+# Travis-Check      : True
+# Script License    : Apache License, Version 2.0
+# Maintainer        : Md. Shafi Hussain <Md.Shafi.Hussain@ibm.com>
+#
+# Disclaimer        : This script has been tested in root mode on given
+# ==========          platform using the mentioned version of the package.
+#                     It may not work as expected with newer versions of the
+#                     package and/or distribution. In such case, please
+#                     contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=vision
+PACKAGE_VERSION=${1:-v0.16.1}
+PACKAGE_URL=https://github.com/pytorch/vision.git
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+PYTHON_VER=${2:-3.9}
+
+WORKDIR=$(pwd)
+
+dnf install -y git jq
+
+# install dependency - pytorch (skip tests)
+PYTORCH_VERSION=${3:-$(curl -sSL https://api.github.com/repos/pytorch/pytorch/releases/latest | jq -r .tag_name)}
+curl -sL https://raw.githubusercontent.com/ppc64le/build-scripts/master/p/pytorch/pytorch_ubi_9.3.sh | sed '/pip install pytest/q' | bash -s $PYTORCH_VERSION $PYTHON_VER
```

Comment:
We should be having instructions to install `pytorch` here in this build script itself, rather downloading the build script.